server/src/email/email-util.es6

```diff
@@ -47,32 +47,35 @@ emailUtil.transporter = nodemailer.createTransport(smtpConfig);
  * @param {boolean} attachments - email contains attachments
  */
 emailUtil.send = (to, subject, body, html = false, attachments = false) => {
-  const bodyAndNoReply = `${body}
+  return new Promise((resolve) => {
+    const bodyAndNoReply = `${body}
 
-    Please do not reply to this message.
-    This email message was sent from a notification-only address that cannot accept incoming email`;
+      Please do not reply to this message.
+      This email message was sent from a notification-only address that cannot accept incoming email`;
 
-  let mailOptions = {
-    from: `"Forest Service online permits" <${vcapConstants.SMTP_USERNAME}>`,
-    to: to,
-    subject: subject,
-    text: bodyAndNoReply // plain text
-  };
-  if (html) {
-    mailOptions.html = html;
-  }
-  if (attachments) {
-    mailOptions.attachments = attachments;
-  }
-  if (vcapConstants.SMTP_HOST) {
-    emailUtil.transporter.sendMail(mailOptions, error => {
-      if (error) {
-        logger.error('NODE_MAILER_SMTP_ERROR', error);
-      } else {
-        logger.info('Email successfully sent');
-      }
-    });
-  }
+    let mailOptions = {
+      from: `"Forest Service online permits" <${vcapConstants.SMTP_USERNAME}>`,
+      to: to,
+      subject: subject,
+      text: bodyAndNoReply // plain text
+    };
+    if (html) {
+      mailOptions.html = html;
+    }
+    if (attachments) {
+      mailOptions.attachments = attachments;
+    }
+    if (vcapConstants.SMTP_HOST) {
+      emailUtil.transporter.sendMail(mailOptions, error => {
+        if (error) {
+          logger.error('NODE_MAILER_SMTP_ERROR', error);
+        } else {
+          logger.info('Email successfully sent');
+        }
+        resolve('bop');
```

Comment:
Just curious: would it be useful to have this method reject if there's an error?